explorer/file_test.cpp

```diff
@@ -30,6 +35,11 @@ class ParseAndExecuteTestFile : public FileTestBase {
                      "testdata/linked_list/typed_linked_list.carbon")) {
         GTEST_SKIP() << "Expensive test to trace";
       }
+    } else {
+      if (is_trace_test) {
+        GTEST_SKIP() << "`trace` tests only check for trace output.";
+        is_trace_test = true;
```

Comment:
This assignment looks redundant.

Reply:
I will remove it in the next commit. Thanks!